Brixcer_Box/src/main.cpp

```diff
@@ -49,20 +54,52 @@ void setup() {
     for (int i = 0; i < KnobCount; i++) {
         KnobControl knob(knobPins[i * 2], knobPins[i * 2 + 1], buttonPins[i]);
         knobs[i] = knob;
-
-        LedControl led(LedChain);
-        leds[i] = led;
+        attachInterrupt(buttonPins[i], buttonClick, FALLING);
+        attachInterrupt(knobPins[i * 2], knobPinTrigger, FALLING);
+        attachInterrupt(knobPins[i * 2 + 1], knobPinTrigger, FALLING);
     }
 }
- 
+
 void loop() {
 
 }
 
-void sendData(int control, int action, int value) {
+int countDigits(int num) {
+    int digits = 0;
+
+    while (num) {
```

Comment:
Will this run forever? Could it get to a point where it doesn't get to 0?

Reply:
No, because it is defined as an int (i.e. rounds to whole number)